sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -128,7 +129,8 @@ public String getDefaultTimezone() {
     return plannerImpl.getDefaultTimezone();
   }
 
-  public void setDefaultTimezone(String timezone) {
+  public void setDefaultTimezone(@UnderInitialization(ZetaSQLQueryPlanner.class) ZetaSQLQueryPlanner this,
```

Comment:
Would ZetaSQLQueryPlanner class be always UnderInitialized while calling setDefaultTimezone ?

Reply:
This can be called at any time, not just initialization. I think it would be possible to delete this function, initialize the timezone in the constructor, and require a new ZetaSQLQueryPlanner to be constructed to change the default.
cc: @robinyqiu

Reply:
@apilloud @robinyqiu Any updates on this ?